skops/card/default_template.md

```diff
@@ -29,6 +29,14 @@ The model plot is below.
 
 {{ model_plot }}
 
+## Evaluation Results
+
+You can find the details about evaluation process and the evaluation results.
+
+{{ eval_methods }}
+
+{{ eval_results | default("[More Information Needed]", true)}}
```

Comment:
these need to be added to the user guide.

Reply:
Added them :)